media/css/m24/flag.scss

```
@@ -20,17 +20,83 @@
 
 .m24-c-flag-subtitle {
     font-size: $text-title-md;
```

```suggestion
    font-size: $text-title-md;
    text-wrap: balance;
    text-wrap-style: balance;
```

Nice spot! The Figma and the demo were different here. Tess had a look and we need to find some kind of middle ground so I'm going to revert the text wrap to match what everyone is used to seeing and we'll tweak after launch.